Jetcaster/tv-app/src/main/java/com/example/jetcaster/tv/ui/component/Background.kt

```diff
@@ -56,3 +89,24 @@ internal fun Background(
             }
     )
 }
+
+@Composable
+internal fun WithBackground(
```

Comment:
This should be named with a noun per our naming API guidance: https://github.com/androidx/androidx/blob/androidx-main/compose/docs/compose-api-guidelines.md#naming-unit-composable-functions-as-entities

How about `EpisodeBackground` or `EpisodeContainer`?